dom.bs

```diff
@@ -1065,6 +1065,15 @@ seen from the definition above, an <a>event listener</a> is a more broad concept
 method, when invoked, must run these steps:
 
 <ol>
+ <li>
+  <p>If the <a>context object</a>'s global object is a {{ServiceWorkerGlobalScope}} object and its
+  associated <a>service worker</a>'s <a>script resource</a>'s <a>has ever been evaluated flag</a> is
+  set, <a>throw</a> a <code>TypeError</code>. [[!SW]]
```

Comment:
I'm a little bit afraid bikeshed won't pick up on these references if I try to build it. Do you happen to have bikeshed installed and could you try? If not, I'll figure it out.